src/Formplot/FileFormat/Vector.cs

```diff
@@ -95,6 +95,17 @@ internal static Vector Deserialize( XmlReader reader )
 			return new Vector( x, y, z );
 		}
 
+		/// <summary>
+		/// Returns the vector with normalized length.
+		/// </summary>
+		public Vector Normalized()
+		{
+			var l = Math.Sqrt( X * X + Y * Y + Z * Z );
+			return l == 0.0
+				? new Vector( 0 )
+				: new Vector( X / l, Y / l, Z / l );
```

Comment:
why not using the operator?
```suggestion
				: this / l;
```